docs-chef-io/content/workstation/install_workstation.md

```diff
@@ -145,7 +145,7 @@ msiexec /q /i MsiPath ADDLOCAL=ALL REMOVE=ChefWSApp
    - Red Hat Enterprise Linux:
 
         ``` bash
-        rpm -Uvh chef-workstation-21.2.278-1.el8.x86_64.rpm
+        yum localinstall chef-workstation-21.2.278-1.el8.x86_64.rpm
```

Comment:
The problem we're going to run into here is that this will only work on RHEL < 8 or RHEL 8 with yum specifically installed

Reply:
RHEL 8 has `yum` installed as a symlink to `dnf`. Red Hat's documentation continues to use `yum` as the recommended userland tool to manage packages.

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/configuring_basic_system_settings/index#installing-packages-with-yum_installing-software-packages

On my Centos Stream 8 host:
```
lrwxrwxrwx. 1 root root 5 Mar 11 11:39 /usr/bin/yum -> dnf-3
```

Reply:
Yeah, I think it's safe to use `yum` here for EL8 and above. No idea if that will change with EL9 and above but for now I think it's OK.